flake.nix

```
@@ -15,6 +15,11 @@
     nix-filter.url = "github:numtide/nix-filter";
   };
 
+  nixConfig = {
+    extra-substituters = "s3://nixcache.chainweb.com?region=us-east-1";
```

```suggestion
    extra-substituters = "https://nixcache.chainweb.com https://cache.iog.io";
```
two things:
- s3 detail is unnecessary
- add iog cache